content/en/docs/4.directory-structure/3.components.md

```diff
@@ -152,8 +152,6 @@ components: {
 }
 ```
 
-Thus, we can use `BaseButton` in the template instead of `Button` without having to change the name of the `Button.vue` file.
```

Comment:
Why removing this @dianhenglau ?

Reply:
Sorry, I accidentally add another commit to this PR. Only the first commit (about the `fetchKey`) is intended, please ignore this commit. I have reset it.